app/server.ts

```diff
@@ -1,16 +1,87 @@
 import next from 'next';
```

Comment:
I decided to refactor out the `express-server.ts` and bring that a into `server.ts` that made getting lightship configuration a bunch simpler.